src/resync.ts

```diff
@@ -68,7 +68,10 @@ export default class Resync {
         log("new state", state)
         this.state.value = state
       }),
-      this.onSource(this.updateMediasession)
+      this.onSource((source?: MediaSourceAny) => {
+        this.updateMediasession(source)
+        if (source) this.seekTo(source.startFrom)
```

Comment:
when does this fire?
for me this seems every client will fire seekTo when the source is changed.